src/bin/lpython.cpp

```diff
@@ -817,6 +817,13 @@ int link_executable(const std::vector<std::string> &infiles,
                 std::cout << "The command '" + cmd + "' failed." << std::endl;
                 return 10;
             }
+            if (outfile == "a.out") {
+                err = system("./a.out");
+                if (err) {
+                    std::cout << "The command './a.out' failed." << std::endl;
+                    return 10;
```

Comment:
Does this work on Windows as well?

Reply:
This doesn't work on Windows. I wonder why?

Reply:
Updated, Now it works on windows as well.